app/helpers/breadcrumb_helper.rb

```diff
@@ -0,0 +1,25 @@
+module BreadcrumbHelper
+  def breadcrumb_part_for_previous_page
+    path, title = path_and_title_from_referer
+
+    content_tag(:li) do
+      content_tag(:a, href: path) do
+        title
+      end
+    end
+  end
+
+  private
+
+  def path_and_title_from_referer
+    return request_uri, 'Search' if request_uri.start_with?(search_appointments_path)
```

Comment:
Stylistically i'm not a huge fan of these stacked returns, I wonder if building up a case statement with the paths interpolated into regex would be neater, also more defensive in that it could fall through to a catch-all, else case. This is nit-picky though and doesn't block you from merging.